CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ if(NOT EXISTS ".mason/mason.cmake")
         WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
 endif()
 
+if(NOT EXISTS "mapbox-gl-js/package.json")
+   execute_process(
+        COMMAND git submodule update --init mapbox-gl-js
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
```

Comment:
I wasn't able to get this to work. It looks to me like the `GIT_SUBMODULES` option is for initializing submodules of the external project, rather than using a submodule as the method to obtain the external project itself.